types/level-errors/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for level-errors 3.0
+// Project: https://github.com/Level/errors
+// Definitions by: Junxiao Shi <https://github.com/yoursunny>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class LevelUPError extends Error {}
+
+export class InitializationError extends LevelUPError {}
+
+export class OpenError extends LevelUPError {}
+
+export class ReadError extends LevelUPError {}
+
+export class WriteError extends LevelUPError {}
+
+export class NotFoundError extends LevelUPError {
+    readonly notFound: true;
+    readonly status: 404;
+}
+
+export class EncodingError extends LevelUPError {}
+
+interface Errors {
+    LevelUPError: typeof LevelUPError;
+    InitializationError: typeof InitializationError;
+    OpenError: typeof OpenError;
+    ReadError: typeof ReadError;
+    WriteError: typeof WriteError;
+    NotFoundError: typeof NotFoundError;
+    EncodingError: typeof EncodingError;
+}
+
+declare const errors: Errors;
+
+export default errors;
```

Comment:
this is wrong - its not a default export. its cjs https://github.com/Level/errors/blob/master/errors.js

Reply:
I see levelup definitions also end with `export default` despite being cjs. Can you explain why?

Reply:
Probably my mistake. @MeirionHughes is right though, these should be cjs exports.

Reply:
some level-* module runtimes do have a 'default' field export that mimics es6 default, but not all (most are not). I wasn't the one to add those runtime fake exports. 

As is, if you import level-errors directly, the typings would be wrong and it would run-time error; but your design-time typings would be fine. I'd suggest `import * as errors from 'level-errors` in level-abstractdown, and `module.exports = errors` in level-errors.